File: app/src/main/java/com/kelsos/mbrc/views/MiniControlView.java
package com.kelsos.mbrc.views;

import android.graphics.Bitmap;
import android.support.annotation.Nullable;

import com.kelsos.mbrc.annotations.PlayerState.State;
import com.kelsos.mbrc.domain.TrackInfo;

public interface MiniControlView extends BaseView {
  void updateCover(@Nullable Bitmap cover);

  void updateTrackInfo(TrackInfo trackInfo);

  void updateState(@State String state);
}

File: app/src/main/kotlin/com/kelsos/mbrc/views/MiniControlView.kt
package com.kelsos.mbrc.views

import com.kelsos.mbrc.annotations.PlayerState.State
import com.kelsos.mbrc.domain.TrackInfo
import com.kelsos.mbrc.mvp.BaseView

interface MiniControlView : BaseView {
  fun updateCover(cover: String)

  fun updateTrackInfo(trackInfo: TrackInfo)

  fun updateState(@State state: String)
}
